### PR TITLE
epsxe: init at 2.0.5

### PR DESCRIPTION
**Pull request by lukateras**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/misc/emulators/epsxe/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    install -Dt $out/bin epsxe
+    patchelf \
+      --set-interpreter $(cat ${stdenv.cc}/nix-support/dynamic-linker) \
+      --set-rpath ${stdenv.lib.makeLibraryPath buildInputs} \
+      $out/bin/epsxe
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = http://epsxe.com/;
+    license = licenses.unfree;
```
Comment (Mic92) [inline pkgs/misc/emulators/epsxe/default.nix]: Can you add a description?

Comment (yegortimoshenko) [reply pkgs/misc/emulators/epsxe/default.nix]: Definitely. In a few minutes.

Comment (yegortimoshenko) [reply pkgs/misc/emulators/epsxe/default.nix]: Done.

File: pkgs/misc/emulators/epsxe/default.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    SDL
+    SDL_ttf
+    stdenv.cc.cc.lib
+    zlib
+  ];
+
+  dontStrip = true;
+  
+  installPhase = ''
+    mv epsxe_* epsxe || true
```
Comment (Mic92) [inline pkgs/misc/emulators/epsxe/default.nix]: Why should this fail?

Comment (yegortimoshenko) [reply pkgs/misc/emulators/epsxe/default.nix]: On i686 Linux, the executable is already called `epsxe`.

Comment (Mic92) [reply pkgs/misc/emulators/epsxe/default.nix]: let's make it explicit

**Comment by yegortimoshenko** [comment]
Thanks a lot!

**Pull request merged**